README.md

```diff
@@ -20,10 +20,9 @@ This component helps you prevent the leakage of secrets even before you push the
 *Prerequisite*
 
 - Java 11
-- maven
 
 ```shell
-mvn clean install
+./gradlew build
```

Comment:
This line is OK, but the next code snippets needs to be updated before merge

Reply:
Updated instructions for runnings tests; code generation hasn't been migrated yet